### DISCUSSION TITLE
G71 file format

### DISCUSSION THREAD

**Discussion by idolpx** [abbruzze/kernal64#20] [Q&A] [answered]
I didn't know it was a real format.  Are there some sample G71 disk image files?
Is it just 2x G64 files concatenated?  Is there a file format spec I can get?  Very curious.

**Comment by abbruzze** [comment, marked as answer]
It was the same for me!
Take a look at https://vice-emu.sourceforge.io/vice_17.html#SEC359
As an example go to https://sourceforge.net/p/vice-emu/bugs/_discuss/thread/8168367d25/6314/attachment/IAmTheC128-35trk.g71

**Reply by idolpx** [reply]
Awesome... thanks.  :)
